src/cpu/primitive_attr_postops.cpp

```diff
@@ -184,8 +183,6 @@ ref_post_ops_t::ref_post_ops_t(const post_ops_t &po, bool skip_sum)
             eltwise_po_.emplace_back(e.eltwise);
         } else if (po_.contain(primitive_kind::binary, idx)) {
             binary_po_.emplace_back(e.binary);
-        } else if (po_.contain(primitive_kind::depthwise, idx)) {
```

Comment:
Why removing this post ops? ref pp doesn't use? only jit_pp would use?

Reply:
I brought this check back.